src/main/java/mate/jdbc/util/ConnectionUtil.java

```diff
@@ -4,29 +4,32 @@
 import java.sql.DriverManager;
 import java.sql.SQLException;
 import java.util.Properties;
+import java.util.Scanner;
 
 public class ConnectionUtil {
-    private static final String URL = "YOUR DATABASE URL";
-    private static final String USERNAME = "YOUR USERNAME";
-    private static final String PASSWORD = "YOUR PASSWORD";
-    private static final String JDBC_DRIVER = "YOUR DRIVER";
+    private static final String URL = "jdbc:mysql://localhost:3306/taxi_service";
+    private static String USERNAME;
+    private static final String PASSWORD;
```

Comment:
if you use scanner there is no reason to name this fields as constants
```suggestion
    private static String username;
    private static String password;
```